src/libs/actions/User.js

```diff
@@ -194,6 +194,39 @@ function setSecondaryLoginAndNavigate(login, password) {
     });
 }
 
+/**
+ * Adds a secondary login to a user's account
+ *
+ * @param {String} login
+ * @param {String} password
+ * @returns {Promise}
+ */
+function setSecondaryLogin(login, password) {
+    Onyx.merge(ONYXKEYS.ACCOUNT, {...CONST.DEFAULT_ACCOUNT_DATA, loading: true});
+
+    return API.User_SecondaryLogin_Send({
+        email: login,
+        password,
+    }).then((response) => {
+        if (response.jsonCode === 200) {
+            const loginList = _.where(response.loginList, {partnerName: 'expensify.com'});
+            Onyx.merge(ONYXKEYS.USER, {loginList, error: ''});
+            return;
+        }
+
+        let error = lodashGet(response, 'message', 'Unable to add secondary login. Please try again.');
```

Comment:
This message should be translated, also below. Why is there a default value here?

Reply:
I just copy paste the content from `setSecondaryLoginAndNavigate`. 
I need `setSecondaryLogin` but I don't need the navigate part. 
I had thought about adding parameter `shouldNavigateToSettingProfile` to `setSecondaryLoginAndNavigate`, but I think it will be not good/clean.
I am not sure, so I just make `setSecondaryLogin` function without navigate part.

Reply:
> I just copy paste the content from setSecondaryLoginAndNavigate.

That's okay but we need to clean both functions. 
Let's use error codes to detect which error has occured, and translate them.

Reply:
@rushatgabhane after some testings I found that the current code doesn't detect error by JSON code because one JSON code for example 666 can have  many different error messages. What should I do here?

Reply:
@rushatgabhane for now, How about make a translation entry for this error message?